Lambda.dfy(24,11): Error: assertion violation

Dafny program verifier finished with 3 verified, 1 error
